core/unit_test/TestTaskScheduler.hpp

```diff
@@ -873,5 +871,7 @@ KOKKOS_IMPL_DISABLE_DEPRECATED_WARNINGS_POP()
 #undef KOKKOS_TEST_WITH_SUFFIX
 #undef KOKKOS_PP_CAT_IMPL
 
+KOKKOS_IMPL_DISABLE_DEPRECATED_WARNINGS_POP()
```

Comment:
As I said earlier we don't really need to pop them in test source files.
Anyway not asking to change.